### ISSUE TITLE
Install broken (dependency download down)

### ISSUE THREAD

**Issue by MadMan247** [YunoHost-Apps/umami_ynh#56]
### Describe the bug

Installation Fails

### Context

- Hardware:  Raspberry Pi at home 4B
- YunoHost version: 11.2.11.2
- I have access to my server: Through SSH | through the webadmin 
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version/branch: 2.11.2~ynh1

### Steps to reproduce

attempt to install application

### Expected behavior

I expected it to install

### Logs

I assure you that my internet access is sufficient, despite the logs.

I wanted to provide the log mentioned in the error log, but it gets nuked with the uninstallation process
`2024-04-19 20:14:23,444: DEBUG - info If you think this is a bug, please open a bug report with the information provided in "/var/www/umami/build/yarn-error.log".`
![image](https://github.com/YunoHost-Apps/umami_ynh/assets/54383546/233051c1-f7ea-4731-aa9d-394577ff6b9e)

https://paste.yunohost.org/raw/gujayimazu

**Comment by MadMan247** [comment]
This has been resolved; I assume the dependent site has been restored.

**Comment by MadMan247** [comment]
Successful install: https://paste.yunohost.org/raw/ezudizalak

Well, I started to get a JSON error when attempting to log in, so I tried to uninstall and reinstall it... so much for that. 
https://paste.yunohost.org/raw/oyezifunaf

**Comment by orhtej2** [comment]
I mean there's not much I can do about npm registry timing out 🤷 
What error were you getting when logging in again?

**Comment by MadMan247** [comment]
I apologize for the delayed response.
As I had uninstalled the umami instance that gave me the json error, I can not tell you with 100% accuracy, however, if memory serves me right, it was along the lines of "JSON.parse, error no index 0 of array 0" and would thus not let me log in. The only configured user at the time was admin, so I had no way to double check if it was just that user.

Some how, I managed to get it installed, but I no longer believe it to be a dependency issue; manually going to the link on a desktop computer successfully downloaded the package, even though the installation failed allegedly due to a timeout. Also, the status page showed no downtime or outages. (https://status.npmjs.org/)

Looking it up, I found many similar issues with yarn packages, such as this: https://github.com/yarnpkg/yarn/issues/8948
and this: https://github.com/yarnpkg/yarn/issues/8242 (though this suggests errors due to disk speed-- I encountered this on a brand new SSD (over USB 3.0) for my Pi 4)
